src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -0,0 +1,672 @@
+<?php
+/**
+ * Interactivity API: WP_Interactivity_API class.
+ *
+ * @package WordPress
+ * @subpackage Interactivity API
+ */
+
+/**
+ * Class used to process the Interactivity API in the server.
+ */
+class WP_Interactivity_API {
+	/**
+	 * Holds the mapping of directive attribute names to their processor methods.
+	 *
+	 * @since 6.5.0
+	 * @var array
+	 */
+	private static $directive_processors = array(
+		'data-wp-interactive' => 'data_wp_interactive_processor',
+		'data-wp-context'     => 'data_wp_context_processor',
+		'data-wp-bind'        => 'data_wp_bind_processor',
+		'data-wp-class'       => 'data_wp_class_processor',
+		'data-wp-style'       => 'data_wp_style_processor',
+		'data-wp-text'        => 'data_wp_text_processor',
+	);
+
+	/**
+	 * Holds the initial state of the different Interactivity API stores.
+	 *
+	 * This state is used during the server directive processing. Then, it is
+	 * serialized and sent to the client as part of the interactivity data to be
+	 * recovered during the hydration of the client interactivity stores.
+	 *
+	 * @since 6.5.0
+	 * @var array
+	 */
+	private $state_data = array();
+
+	/**
+	 * Holds the configuration required by the different Interactivity API stores.
+	 *
+	 * This configuration is serialized and sent to the client as part of the
+	 * interactivity data and can be accessed by the client interactivity stores.
+	 *
+	 * @since 6.5.0
+	 * @var array
+	 */
+	private $config_data = array();
+
+	/**
+	 * Gets and/or sets the initial state of an Interactivity API store for a
+	 * given namespace.
+	 *
+	 * If state for that store namespace already exists, it merges the new
+	 * provided state with the existing one.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $store_namespace The unique store namespace identifier.
+	 * @param array  $state           Optional. The array that will be merged with the existing state for the specified
+	 *                                store namespace.
+	 * @return array The current state for the specified store namespace.
```

Comment:
Same suggestion here as the global state function:

```suggestion
	 * @return array The state for the specified store namespace. This will be the updated state if a $state argument was provided.
```

Reply:
Thanks! Done in https://github.com/WordPress/wordpress-develop/pull/5953/commits/7aaf308be82ac20be5e182d4a351a626b249e6f2.